IS0402Test.py

```diff
@@ -816,6 +819,243 @@ def test_30(self):
         else:
             return test.FAIL("Version > 1 not supported yet.")
 
+    def test_31(self):
+        """Query API sends correct websocket event messages for SYNC, ADD, MODIFY and DELETE"""
```

Comment:
Should we aim for consistent terminology with the spec [WebSocket protocol](https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.2.x/docs/4.2.%20Behaviour%20-%20Querying.md#websocket-protocol), i.e. "sync" (or "unchanged"), "added", "modified" and "removed"?